ci-operator/config/openshift-psap/special-resource-operator/openshift-psap-special-resource-operator-master.yaml

```diff
@@ -1,22 +1,15 @@
 build_root:
   image_stream_tag:
-    name: release
-    namespace: openshift
-    tag: golang-1.17
+    name: builder
+    namespace: ocp
+    tag: rhel-8-golang-1.18-openshift-4.11
 images:
 - dockerfile_path: Dockerfile
   to: special-resource-operator
 - dockerfile_literal: |
     FROM src
     RUN ["yum", "update", "-y"]
     RUN ["yum", "install", "-y", "ShellCheck"]
-    RUN mkdir /tmp/helm && \
```

Comment:
I think I already tried to remove it but it made the e2e rehears test to fail. Can't remember what was the issue.

Reply:
Maybe that was d/s only.